setup_template.py

```diff
@@ -70,16 +168,18 @@ def main():
             content = content.replace("Reinder Vos de Wael", username)
 
             if not content == content_before:
-                print(f"Updating {file.relative_to(dir_repo)}")
-                with open(file, "w") as f:
+                print(f"Updating {file.relative_to(DIR_REPO)}")
+                with open(file, "w", encoding="utf-8") as f:
                     f.write(content)
 
-    dir_module = dir_repo / "src" / "APP_NAME"
+    replace_license(repo_license)
+
+    dir_module = DIR_REPO / "src" / "APP_NAME"
     if dir_module.exists():
         dir_module.rename(dir_module.parent / module_name)
 
     # Remove this file
-    print(f"Removing {file_self.name}")
+    print(f"Removing {__file__}")
```

Comment:
This also prints the full path this way

Reply:
Replaced with a catch-all statement as more files are removed now.